domains/transfer/Transfer.API/API/Claiming/Api/Dto/Response/ClaimSubjectHistoryEntriesDto.cs

```diff
@@ -0,0 +1,6 @@
+using System.Collections.Generic;
+using API.Claiming.Api.Models;
+
+namespace API.Claiming.Api.Dto.Response;
+
+public record ClaimSubjectHistoryEntriesDto(List<ClaimSubjectHistoryEntryDto> History);
```

Comment:
Does this follow the rules of pagination in the API Guidelines? Also, it differs from the same history response for Transfer Agreement

Reply:
It doesnt follow pagination. 
Maybe it should.
Regarding the response type its purpose is to serve the frontend and they havent specifiec the needs yet.do you refer to the enum on the auditaction?

Reply:
I was referred to the name of the property for the array. In the the API guidelines, it is called `items`: https://energinet.atlassian.net/wiki/spaces/EnergyOrigin/pages/460947457/API+Guidelines#Must-support-pagination. For this endpoint, it is not called `items`, but is called `history`. For TA history, it is called `result`.

Reply:
corrected to items